packages/forms/.storybook/config.js

```diff
@@ -40,4 +36,10 @@ addDecorator(
 );
 addDecorator(withA11y);
 addDecorator(withFormLayout);
-configure(loadStories, module);
+configure(
+	[
+		require.context('../src', true, /\.stories\.js$/),
+		require.context('../stories-core', false, /index\.js$/),
+	],
+	module,
+);
```

Comment:
Let's not stay in the past :D 
The stories for the widgets will be located next to the widget. So we still load `stories-core` folder for the old stories, but we load the `.stories.js` files too